grpc.go

```diff
@@ -656,7 +656,10 @@ func (a *agentGRPC) SignalProcess(ctx context.Context, req *pb.SignalProcessRequ
 	// processes inside the container.
 	// If the process is the container process, let's use the container
 	// API for that.
-	if req.ExecId == "" {
+	// Frozen processes don't receive signals, but we'll do an exception
+	// because of we don't know why docker containers can, see https://github.com/moby/moby/issues/37137.
```

Comment:
The change looks fine to me, but I really don't like this kind of comment.
The simple explanation is that all processes have to be thawed before we can signal them, and that's why we need to make sure we use `allProcesses` flag as `true` for this specific use case.